### PR TITLE
Fix/level 2 items

### PR DESCRIPTION
**Pull request by touchRED**
Notes:

- Removes page-level fields for social sharing images, descriptions, keywords – opting to use site-level defaults instead
- Reverts about page font style, using Gotham instead of Caslon
- Changes Home Page explorer card graphic
- Includes higher res (2x) imagery for research topics

Important: for these new 2x images, I opted to run them through compressor.io, which is lossy. If we want the full 2x images with no compression, just lmk and I can reupload.

### CODE REVIEW

File: src/components/atoms/seo.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import Helmet from 'react-helmet'
 import { useStaticQuery, graphql } from 'gatsby'
 
 function SEO({ meta }) {
-  // console.log('SEO, ', meta)
+  console.log('SEO, ', meta)
```
Comment (Lane) [inline src/components/atoms/seo.js]: can we drop this? adds some noise to logs

File: src/components/atoms/seo.js [hunk 2]
```diff
@@ -37,20 +37,16 @@ function SEO({ meta }) {
     `
   )
 
-  const description = meta.description
-    ? meta.description
-    : site.siteMetadata.description
+  const description = site.siteMetadata.description
```
Comment (Lane) [inline src/components/atoms/seo.js]: sorry, might have been some confusion on my request here.  we still want to keep the option for page level meta descriptions, but if there is not one specified in the page frontmatter it should use the site default.  i think this is how it was configured before this change, and we'd just need to remove the descriptions from the frontmatter to make this happen?

**Review by Lane — CHANGES_REQUESTED** [review]
thanks, looks good! just one adjustment on the page description metadata.

**Comment by touchRED** [comment]
@Lane uncompressed images are good to go, also removed that SEO log

File: src/components/atoms/seo.js (hunk 2 above)
Comment (touchRED) [reply src/components/atoms/seo.js]: @Lane should I leave that block as is? with:
`
const description = meta.description    ? meta.description    : site.siteMetadata.description`

also, should I take the description field out of the frontmatter entirely? just wanna make sure i understand what you're suggesting

Comment (Lane) [reply src/components/atoms/seo.js]: yep, leave the block as it was and then remove generic `Page description` values from frontmatter (but leave the key)

e.g. 

```
description: Page description
```

becomes:
```
description:
```

without a value specified, it should use site default, but if they enter a value it should use that.

**Comment by Lane** [comment]
@touchRED thanks! can you drop the placeholder page descriptions from the page frontmatter so it uses the site default?

**Comment by touchRED** [comment]
✅

**Pull request merged**